tests/test_response.py

```diff
@@ -85,6 +86,49 @@ def test_response_body_passes_through_when_base64(self) -> None:
             (666, False),
         ],
     )
-    def test_response_is_ok(self, code: int, outcome: bool) -> None:
+    def test_response_ok(self, code: int, outcome: bool) -> None:
```

Comment:
Please, follow the naming convention that we agreed on a long time ago.

Here and in the below tests.